core/src/main/res/values/strings.xml

```diff
@@ -388,6 +392,7 @@
     <string name="pref_mobileUpdate_episode_download">Episode download</string>
     <string name="pref_mobileUpdate_streaming">Streaming</string>
     <string name="user_interface_label">User Interface</string>
+    <string name="user_interface_sum">Appearance, subscription order, lockscreen..</string>
```

Comment:
I have not seen an app that uses ellipses in the end of settings descriptions. The main Android settings app does not use them, too. I would prefer to not add them.

Reply:
My favorite Android podcast app is BeyondPod and they use ellipses, but I will remove them, they add no real value except to convey there is more...